Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN curl -L ${NYCDB_REPO}/archive/${NYCDB_REV}.zip > nycdb.zip \
   && pip install -e .
 
 ARG WOW_REPO=https://github.com/justFixNYC/who-owns-what
-ARG WOW_REV=ff23fc0065d7deca3a0f89cdf2bf234b0bb68f62
+ARG WOW_REV=a2f4d2b1416ba7d9780b90b31202f3833efbae8d
```

Comment:
Ack!  Wait, is there a particular reason you changed the `WOW_REV` here?  It looks like the rev was https://github.com/JustFixNYC/who-owns-what/commit/ff23fc0065d7deca3a0f89cdf2bf234b0bb68f62, which was made on May 25, corresponding to the changes made in #49, while the revision you've changed it to is https://github.com/JustFixNYC/who-owns-what/commit/a2f4d2b1416ba7d9780b90b31202f3833efbae8d, which dates from March 12...

Reply:
No I didn't mean to do that at all, sorry about that, something must have gotten mixed up with the history on my fork. I just changed it back to to what is is on master right now.